src/translations/en_DK.json

```diff
@@ -205,13 +205,13 @@
   "START_PAGE_TITLE": "Get a price quote | Hedvig",
   "SIDEBAR_INSURANCE_TYPE_STUDENT_BRF": "Home Contents Student",
   "SIDEBAR_INSURANCE_TYPE_STUDENT_RENT": "Rental Student",
-  "SIDEBAR_INSURANCE_TYPE_NO_CONTENTS_OWN": "Contents insurance",
+  "SIDEBAR_INSURANCE_TYPE_NO_CONTENTS_OWN": "Home Contents Insurance",
```

Comment:
Optional if you want to change this now since I don't think anyone have told you about this :) But when we have market specific textkeys, it's good to use `[VOID]` for the other languages and markets in Lokalise. That means it's exported as empty string, but counts as translated in the Lokalise.

Reply:
Oh yeah, @styrelius showed me that! I added it for the DK/SE strings but the English were all filled in. But I can go back and replace them with `[VOID]` - makes sense!